playbooks/templates/.env.j2

```diff
@@ -219,11 +231,8 @@ STRIPE_WEBHOOK_SECRET={{ webportal_common_config.stripe_webhook_secret | default
 API_HOST=10.10.10.10
 SKYNET_ACCOUNTS_HOST=10.10.10.70
 SKYNET_ACCOUNTS_PORT=3000
-
-# Sia api password. 
-#
-# REQUIRED: manually defined by user, or Ansible automatically generated 
-SIA_API_PASSWORD={{ webportal_server_config.sia_api_password }}
+BLOCKER_HOST=blocker
```

Comment:
Karol are previously removed these because he hard coded them into the docker compose file. https://github.com/SkynetLabs/skynet-webportal/blob/master/docker-compose.blocker.yml#L12

Is there a reason you are adding them back?

Reply:
Removed.